<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>518</width>
    <height>765</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>421</width>
      <height>711</height>
     </rect>
    </property>
    <property name="toolTip">
     <string/>
    </property>
    <property name="currentIndex">
     <number>2</number>
    </property>
    <widget class="QWidget" name="Profile">
     <property name="accessibleName">
      <string/>
     </property>
     <attribute name="title">
      <string>Profile</string>
     </attribute>
     <widget class="QLabel" name="ProfileLoginLabel">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>20</y>
        <width>41</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Login</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="ProfileLoginLineEdit">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>10</y>
        <width>251</width>
        <height>41</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="ProfileFirstNameLabel">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>70</y>
        <width>61</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>First name</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="ProfileFirstNameLineEdit">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>60</y>
        <width>251</width>
        <height>41</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="ProfileLastNameLabel">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>120</y>
        <width>61</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Last name</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="ProfileLastNameLineEdit">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>110</y>
        <width>251</width>
        <height>41</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="ProfileStatusLabel">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>170</y>
        <width>41</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Status</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="ProfileStatusTextEdit">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>160</y>
        <width>251</width>
        <height>192</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="ProfileChangePasswordButton">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>360</y>
        <width>121</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Change password</string>
      </property>
     </widget>
     <widget class="QPushButton" name="ProfileApplyButton">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>360</y>
        <width>91</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Apply</string>
      </property>
     </widget>
     <widget class="QPushButton" name="ProfileCancelButton">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>360</y>
        <width>91</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Cancel</string>
      </property>
     </widget>
     <widget class="QLabel" name="ProfileSuccessLabel">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>420</y>
        <width>351</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="Chats">
     <attribute name="title">
      <string>Chats</string>
     </attribute>
     <widget class="QTextEdit" name="ChatsTextEdit">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>489</y>
        <width>341</width>
        <height>81</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="ChatsSendButton">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>490</y>
        <width>41</width>
        <height>81</height>
       </rect>
      </property>
      <property name="text">
       <string>-&gt;</string>
      </property>
     </widget>
     <widget class="QPushButton" name="ChatsBackButton">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>570</y>
        <width>101</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Back</string>
      </property>
     </widget>
     <widget class="QListWidget" name="ChatsListWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>381</width>
        <height>481</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="ChatsConfigureChatButton">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>570</y>
        <width>161</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Configure the chat</string>
      </property>
     </widget>
     <widget class="QPushButton" name="ChatsCreateChatButton">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>570</y>
        <width>121</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Create a chat</string>
      </property>
     </widget>
     <widget class="QLabel" name="ChatsSuccessLabel">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>620</y>
        <width>381</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="Contacts">
     <attribute name="title">
      <string>Contacts</string>
     </attribute>
     <widget class="QTableWidget" name="ContactsTableWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>0</y>
        <width>381</width>
        <height>441</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="AddContactButton">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>460</y>
        <width>151</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Add a new contact</string>
      </property>
     </widget>
     <widget class="QPushButton" name="DeleteContactButton">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>460</y>
        <width>151</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Delete a contact</string>
      </property>
     </widget>
     <widget class="QLabel" name="ContactsSuccessLabel">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>610</y>
        <width>381</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QTextEdit" name="ContactsTextEdit">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>510</y>
        <width>381</width>
        <height>151</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>518</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="Menu">
    <property name="title">
     <string>Menu</string>
    </property>
    <addaction name="actionReconnect"/>
    <addaction name="actionRefresh"/>
   </widget>
   <addaction name="Menu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionReconnect">
   <property name="text">
    <string>Reconnect</string>
   </property>
  </action>
  <action name="actionRefresh">
   <property name="text">
    <string>Refresh</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
